ev3dev2/motor.py

```diff
@@ -1739,18 +1834,6 @@ def _unpack_speeds_to_native_units(self, left_speed, right_speed):
             right_speed
         )
 
-    def on_for_rotations(self, left_speed, right_speed, rotations, brake=True, block=True):
```

Comment:
I moved this to after on_for_degrees() because on_for_rotations() call on_for_degrees()